static/locales/en-US.yaml

```diff
@@ -830,6 +830,8 @@ Video:
   Streamed on: Streamed on
   Started streaming on: Started streaming on
   Sponsor Block category:
+    Show Original Details: Show Original Details
+    Show Modified Details: Show Modified Details
```

Comment:
Could you maybe move these strings to `Video.DeArrow` or something like that instead, as they aren't SponsorBlock segment categories. Sorry that I didn't notice this earlier, as I could have suggested it with my other code suggestions and saved you time 🫣.

Reply:
No worries